server/src/load_data.py

```diff
@@ -193,6 +193,13 @@ async def load_all_data():
         except Exception as e:
             print("Index creation failed. Index already exists")
             # traceback.print_exc()
+
+        try:
```

Comment:
Please remove this. Index is not required for caching. Since it will be queried only by key.